tags.go

```diff
@@ -34,10 +34,12 @@ type TagsService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/tags.html
 type Tag struct {
-	Commit  *Commit      `json:"commit"`
-	Release *ReleaseNote `json:"release"`
-	Name    string       `json:"name"`
-	Message string       `json:"message"`
+	Commit    *Commit      `json:"commit"`
+	Release   *ReleaseNote `json:"release"`
+	Name      string       `json:"name"`
+	Message   string       `json:"message"`
```

Comment:
Not sure how it's handled in general here, but the `message` is actually optional and maybe `null` (according to docs examples).

Reply:
Firstly, this case doesn't have tests with this fields. Secondly, I add only protected and target fields. I can realize additional featches but in this moment it's overhead)

Reply:
The tests comment was a general "review" comment, not for this particular line or `message` field.

Tests are here which you'd be able to extend: https://github.com/xanzy/go-gitlab/blob/b5c61e10d5a83e626c3b33823880b0274b393e2d/tags_test.go#L42